scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryTypes.scala

```diff
@@ -53,7 +53,7 @@ object Table {
     override lazy val spec: String = BigQueryHelpers.toTableSpec(ref)
   }
   final case class Spec(spec: String) extends Table {
-    override lazy val ref: GTableReference = BigQueryHelpers.parseTableSpec(spec)
+    override val ref: GTableReference = BigQueryHelpers.parseTableSpec(spec)
```

Comment:
@nevillelyh should we make line 53 eager as well?

Reply:
We can but the impl won't throw any exception so it doesn't make a difference anyway?

Reply:
yeah it kinda makes that it doesn't throw; let's keep it as is